standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/TestHiveMetaStore.java

```diff
@@ -3428,4 +3432,44 @@ public void testUpdatePartitionStat_doesNotUpdateStats() throws Exception {
     m.invoke(hms, tbl, part, false, null);
     verify(wh, never()).getFileStatusesForLocation(part.getSd().getLocation());
   }
+
+  @Test(expected = InvalidOperationException.class)
```

Comment:
Also, Is it a positive test case or negative? I see that there is an expected error but we are doing the positive validations too. We should break this into two, right?

Reply:
Divided the test in negative and positive